account_chatter/views/account_account_views.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+
+    <record id="view_account_form_show_chatter" model="ir.ui.view">
```

Comment:
Since this is now a standard inheritance (without groups), ID should match inherited view